From 175f16dc6f41d7a7f78a9e75019d85a6e13c8f49 Mon Sep 17 00:00:00 2001
From: Kevin Hao <kexin.hao@windriver.com>
Date: Wed, 9 May 2018 09:36:57 +0800
Subject: [PATCH 1/4] net: fman: Fix the misleading indentation warning

Fixes the following misleading indentation warning introduced by the
commit a2381b284505 ("fmd: SDK DPAA 1.x FMan driver"):
  drivers/net/ethernet/freescale/sdk_fman/Peripherals/FM/fman.c:461:5: warning: this 'if' clause does not guard... [-Wmisleading-indentation]
       if ((port_id > 63) || (port_id < 1))
       ^~

Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/net/ethernet/freescale/sdk_fman/Peripherals/FM/fman.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/net/ethernet/freescale/sdk_fman/Peripherals/FM/fman.c b/drivers/net/ethernet/freescale/sdk_fman/Peripherals/FM/fman.c
index a41ecd0..a141a0e 100755
--- a/drivers/net/ethernet/freescale/sdk_fman/Peripherals/FM/fman.c
+++ b/drivers/net/ethernet/freescale/sdk_fman/Peripherals/FM/fman.c
@@ -458,8 +458,8 @@ uint8_t fman_get_num_of_tasks(struct fman_bmi_regs *bmi_rg, uint8_t port_id)
 {
 	uint32_t tmp;
 
-    if ((port_id > 63) || (port_id < 1))
-        return 0;
+	if ((port_id > 63) || (port_id < 1))
+		return 0;
 
 	tmp = ioread32be(&bmi_rg->fmbm_pp[port_id - 1]);
 	return (uint8_t)(((tmp & BMI_NUM_OF_TASKS_MASK) >>
@@ -470,8 +470,8 @@ uint8_t fman_get_num_extra_tasks(struct fman_bmi_regs *bmi_rg, uint8_t port_id)
 {
 	uint32_t tmp;
 
-    if ((port_id > 63) || (port_id < 1))
-        return 0;
+	if ((port_id > 63) || (port_id < 1))
+		return 0;
 
 	tmp = ioread32be(&bmi_rg->fmbm_pp[port_id - 1]);
 	return (uint8_t)((tmp & BMI_NUM_OF_EXTRA_TASKS_MASK) >>
-- 
1.8.5.2.233.g932f7e4

